hw/ip/otbn/data/insns.yml

```diff
@@ -113,7 +113,8 @@ encoding-schemes: enc-schemes.yml
 #             source; "grd" for a destination.
 #
 #  wdr:       The name of a wide register. Syntax "wrs" for a source;
-#             "wrd" for a destination.
+#             "wrd" for a destination; "wru" for a register that is updated
```

Comment:
I think wrb better describes this, since it is both a source and destination. The word updated doesn't mix so well with source and destination concepts: it is a verb, and the destination is also updated.

Also, this description is confusing at another level: The valid operand types are "grs" and "grd" for general purpose register source and destination, "wrd", "wrs", and (hopefully) "wrb" for wire register source, destination, and "both", and "csr" for csr registers, which are always a source. What I mean is that the lists should lead with the actual type string, and mention the kind of register in the description.

Reply:
Good point about the dubious name. I'm also not a huge fan of "both" (because it doesn't mean anything out of context: "both red and blue?"), but I can't think of anything better, so let's go for wrb for now.

Also, good point about this documentation comment as a whole. I'll file a followup PR that rejigs things to make it a bit clearer.

Reply:
Thanks